### PR TITLE
Parser

### PR DESCRIPTION
**Pull request by samikuikka**
primitive way to create prompt from theme and user prompt.

### CODE REVIEW

File: backend/src/services/promptService.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export const create = (theme: string, input: string): string => {
+    return theme + input;
```
Comment (hmalmsten) [inline backend/src/services/promptService.ts]: Appending of strings should contain space between theme and input

Comment (samikuikka) [reply backend/src/services/promptService.ts]: Good point! I'll go fix this :)

**Review by KorpelainenTommi — APPROVED** [review]
👍

**Review by hmalmsten — APPROVED** [review]
Very nice programming. Quite exquisite indeed. Perfect example of ideal code. Contains the very essence of programming itself to be immortalized for future generations to witness.

**Pull request merged**